### PR TITLE
Expose rpc port on daemon:start

### LINKED ISSUE #159: Expose tendermint port
In order to access some Tendermint features eg: blocks, txs, validators, etc... we should be able to communicate with its API and so expose the RPC port of Tendermint when we start the daemon.

Default port: `26657`

### PR DESCRIPTION
**Pull request by antho1404**
fix https://github.com/mesg-foundation/js-sdk/issues/159

Add `rpc-port` flag to the `daemon:start` command to expose the RPC API

### CODE REVIEW

File: packages/cli/src/commands/daemon/start.ts [hunk 1]
```diff
@@ -32,6 +32,11 @@ export default class Start extends Command {
       default: 26656,
       required: true,
     }),
+    'rpc-port': flags.integer({
```
Comment (NicolasMahe) [inline packages/cli/src/commands/daemon/start.ts:35]: i recommend to include `tendermint` in the name. We already have multiple RPC servers, engine and tendermint, and we may have also one for cosmos modules.

Comment (NicolasMahe) [reply packages/cli/src/commands/daemon/start.ts:35]: maybe `port` could be renamed to `rpc-port`. and then tendermint's port to `tendermint-rpc-port`, etc..

Comment (antho1404) [reply packages/cli/src/commands/daemon/start.ts:35]: Or we could just remove all notion of rpc (that most people might not know) and just have `port` and `tm-port` or `tendermint-port`

Comment (NicolasMahe) [reply packages/cli/src/commands/daemon/start.ts:35]: good idea, i like `tendermint-port`!

Comment (NicolasMahe) [reply packages/cli/src/commands/daemon/start.ts:35]: Could you also add the prometheus port with a similar name?

**Review by NicolasMahe — CHANGES_REQUESTED** [review]
Let's name the flag `tendermint-port`.
Could you also add the prometheus port with a similar name?

**Comment by antho1404** [comment]
After discussion, let's close this PR, this adds too much complexity on the CLI, we need a way to enable/disable the port and at the same time keep the default port so this adds 2 flags for each port we want to expose so 4 flags if we have tendermint + prometheus.

Let's just update the documentation with the commands to type to export these ports and keep the CLI simple for now.

**Pull request closed without merging**